### PR TITLE
[ST] Remove NodePort from the 'weird username tests' for Kafka Bridge

### LINKED ISSUE #7608: Remove un-necessary NodePorts from system test.
In `HttpBridgeKafkaExternalListenersST.java` (https://github.com/strimzi/strimzi-kafka-operator/pull/7526), we have a test case (i.e., testWeirdUsername), which currently uses NodePort. Fortunately, we can remove it, because it is leftover during the refactoring of test clients by Lukas. 

> IIRC reason why we are using external producer etc. was that at the time I was writing those "weird username tests", there were no possibilities to use internal clients - as they didn't allow any username to be longer than some number of characters and it shouldn't contain any special characters. External clients are allowing it.

Test cases currently do the following:
1. Create a bridge consumer to receive messages from the external listener (NodePort)
2. Create a producer to produce messages via the same listener (NodePort)

But we can easily modify 2. point to use internal listeners instead of NodePort.

### PR DESCRIPTION
**Pull request by im-konge**
### Type of change

- Enhancement

### Description

In the past, there was no way how to write "weird username" tests with our internal clients, as they used the Kafka scripts for the producer/consumer, which had limitation in the length of the user name - which is the main part of this test -> testing that everything works with some special characters and longer names. In the external clients (which are just our implementation of the Java Kafka clients) there is not such limitation -> that's why we used them. But after the creation of `test-clients`, there is not such limitation as well. So in the tests it is not needed to keep the external clients.

This PR removes the NodePort from the "weird username tests" for the Kafka Bridge. Together with that, I'm moving the tests into the `HttpBridgeTls` class, because the "external listener" is not a thing after the changes, so the extra class is not needed.

Fixes #7608 

### Checklist

- [ ] Make sure all tests pass

### CODE REVIEW

File: systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeTlsST.java [hunk 1]
```diff
@@ -90,13 +97,167 @@ void testReceiveSimpleMessageTls() {
...
+            .endKafkaClientAuthenticationTls()
+            .withNewTls()
+                .withTrustedCertificates(certSecret)
+            .endTls()
+            .build();
+
+        testWeirdUsername(weirdUserName, new KafkaListenerAuthenticationTls(), bridgeSpec, testStorage);
+    }
+
+    @SuppressWarnings({"checkstyle:MethodLength"})
```
Comment (henryZrncik) [inline systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeTlsST.java:156]: I don't think we actually need this warning suppression with current length.

File: systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeTlsST.java [hunk 2]
```diff
@@ -90,13 +97,167 @@ void testReceiveSimpleMessageTls() {
...
+
+        final KafkaClients kafkaClients = new KafkaClientsBuilder()
+            .withNamespaceName(testStorage.getNamespaceName())
+            .withBootstrapAddress(KafkaResources.tlsBootstrapAddress(testStorage.getClusterName()))
+            .withUsername(weirdUserName)
+            .withMessageCount(testStorage.getMessageCount())
+            .withTopicName(testStorage.getTopicName())
+            .withProducerName(testStorage.getProducerName())
+            .withConsumerName(testStorage.getConsumerName())
+            .build();
```
Comment (henryZrncik) [inline systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeTlsST.java:239]: maybe 

```
final KafkaClients kafkaClients = ClientUtils.getInstantTlsClientBuilder(testStorage)
    .withUsername(weirdUserName)
    .build();
```

**Review by henryZrncik — APPROVED** [review]
Works for me thanks for changes ! Just a little nit.

**Review by Frawless — APPROVED** [review]

**Review by see-quick — APPROVED** [review]

**Pull request merged**